index.html

```diff
@@ -97,7 +100,7 @@ <h1>GAMERS</h1>
     </nav>
     <header class="pb-4">
       <div class="container">
-        <div class="row">
+        <div class="row ">
           <div class="col-md-6 d-flex flex-column text-contorller">
             <h2>PS5 V2 <br />DualShock 5</h2>
             <p>
```

Comment:
Each card content should have paddings to separate the content from the edges, the card content padding is missing and the space between the price and bottom of the card is not like the design